index.js

```diff
@@ -0,0 +1,69 @@
+// global variables
+let searchCityForm = document.getElementById('searchCityForm');
+let errorMessage = document.getElementById('errorMessage');
+let city = document.getElementById('city');
+let temp = document.getElementById('temp');
+let dataWrapper = document.getElementById('dataWrapper');
+let weatherImg = document.getElementById('weatherImg');
+let fah;
+
+
+let convertTemp = (kelvin) => {
```

Comment:
Does convertTemp always receive a value in Kelvin as an argument? Will it ever receive a Celsius or Fahrenheit value?